packages/webcert/src/components/modal/ExpandableCareUnit.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react'
```

Comment:
Also here it feels like the expandable components can be extracted to general, reusable components and separated from the logic of the Units